### PR TITLE
Toggle menu

### PR DESCRIPTION
**Pull request by QueenterJuma**
## What does this PR do?
This PR is raised to create a Portfolio using a Mobile template from Figma
##Description of the task to be completed
 [ x] Build a hamburger menu which can be toggled
 [x] Use a mobile Figma design.
 [x] Use images and backgrounds to enhance the look of the website
 [x] Use the [HTML&CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css)
 
## How should it be tested
Clone the repository
```
git clone https://github.com/QueenterJuma/Portfolio_1/pulls 
```
Navigate to the folder
```
cd repo-name
```
Checkout the branch
```
git checkout branch_accecibility
```

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi Team
I'm sure you put a lot of effort into this project but this PR is invalid because you didn't setup `Eslint` linters for this project 😓  

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-toggleMenu.addEventListener("click", () => {
+toggleMenu.addEventListener('click', () => {
   if (!menuOpen) {
-    toggleMenu.classList.add("open");
+    toggleMenu.classList.add('open');
     menuOpen = true;
   } else {
-    toggleMenu.classList.remove("open");
+    toggleMenu.classList.remove('open');
     menuOpen = false;
```
Comment (Munsa1) [inline script.js:11]: - [ ] The links in the mobile menu should redirect the user to the appropriate section of the mobile view when clicked. Kindly do so. 
For example when the user clicks the `Portfolio` link it should go to the top of the page, when the user clicks the `About Me` page it should take the user to the about me section and when the `Contact` link is clicked it should take the user to the contact form.

image

- [ ] The mobile view should also cover the whole mobile view for user experience unlike the current situation where, if the the user scrolls down, the menu remains at the top. Using a `position: fixed` property on the mobile view parent container fixes this
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @QueenterJuma, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] It is a requirement that we use the newly updated `Readme` file. [Follow this link](https://github.com/microverseinc/readme-template/blob/master/README.md)
#### Your Current Readme File Is Outdated
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

## Hey @egichuhis, @QueenterJuma :wave:

Great work on implementing the requirements. Your project is complete. :blush:
There is nothing else to say about your project. :+1:


## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Linters
✔️ Following requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**